templates/src/graphql/resolvers.js

```diff
@@ -0,0 +1,65 @@
+const FileService = require("../service/file");
+const TripService = require("../service/trip");
+
+const fileService = new FileService();
+const tripService = new TripService();
+
+module.exports = {
+    Query: {
+        presignedUrls: (_, { key }, context) => {
+            const presignedurls = {
+                upload: fileService.getUploadPresignedUrl( key ),
+                download: fileService.getDownloadPresignedUrl( key )
+            };
+            return presignedurls;
+        },
+        userInfo: (_, {}, context) => {
+            return context.user;
+        },
+        status: (_, {}, {}) => {
+            var podName = (process.env.POD_NAME)?process.env.POD_NAME:"zero-node-backend";
+            return {
+                ready: "OK",
+                alive: "OK",
+                podName: podName
+            };
+        },
+        bookedTrips: (_, { }, context) => {

```

Comment:
Can you split the "Trip" resolvers out into a separate file so they are easy to remove when the user wants to use it? I'm not convinced we even need this Trip stuff but if we have it, it's only purpose is as an example so it needs to be easy to fully remove without having to change a bunch of different files.

Reply:
If we do keep it, maybe we should tag all the Trip stuff with a certain comment so it's easy for people to search for?

Reply:
split "Trip" from the default schema and resolver.